src/app.tsx

```diff
@@ -112,7 +115,15 @@ function App(): JSX.Element {
   }, [fetchGlobalState]);
 
   useEffect(() => {
-    new LocalStorageProvider().saveApplicationVersion().then();
+    const checkVersions = async (): Promise<void> => {
+      const latestLocalVersion = CookiesProvider.getCookie("appversion");
```

Comment:
Lepiej enkapsulować tę logikę w `CookiesProviderze`:
zrobić tak żeby on posiadał metodę która nazywa się `getAppVersion` która pod spodem będzie używać tej metody z `getCookie("appverison")`

Motywacja jest taka, że jeśli użyjesz tej metody w innych miejscach, a potem np. zmienisz nazwę cookie pod którym to jest przechowywane - to potrzeba będzie zmienić nazwę tego cookie tylko w tej funkcji, zamiast w każdym miejscu gdzie to używasz jak było by teraz